app/operations/subject_groups/create.rb

```diff
@@ -83,12 +96,27 @@ def build_group_subject
 
     def subject_location_params
       subject_locations = find_subjects_in_order(subject_ids).map(&:locations).flatten
+      external_locations_subject_id_lut = {}
       external_locations = subject_locations.map do |loc|
         extension = File.extname(loc.src).downcase[1..-1]
         mime_type = Mime::Type.lookup_by_extension(extension).to_s
-        { mime_type => "https://#{loc.src}" }
+        media_url = "https://#{loc.src}"
+        # track the subject URL and the originating subject id
+        external_locations_subject_id_lut[media_url] = loc.linked_id
+        # return the mime / url combination for building the media location resources
+        { mime_type => media_url }
+      end
+      Subject.location_attributes_from_params(external_locations).map do |location_param|
```

Comment:
this method is getting pretty big / can probably be broken up - i'll put that on the todo list (feel free to remind me in review as well)